spec/models/tag_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe Tag do
+
+  it "decreases tag_count when a debate is hidden" do
+    debate = create(:debate)
+    tag = create(:tag)
```

Comment:
ok, then maybe adding this information to the commit message? :thinking: 

Yes, when this get merge I can send a PR to improve it. :wink: